proposals/Level-1.md

```diff
@@ -202,27 +180,30 @@ associated with the exception index. These values are popped of the stack and
 are used (along with the corresponding exception tag) to create the
 corresponding exception value. That exception value is then thrown.
 
-When an exception value is thrown, the host VM searches for nearest enclosing
-try block body that execution is in. That try block is called the _catching_ try
-block.
+When an exception value is thrown, the embedder searches for the nearest
+enclosing try block body that execution is in. That try block is called the
+_catching_ try block.
 
 If the throw appears within the body of a try block, it is the catching try
 block.
 
 If a throw occurs within a function body, and it doesn't appear inside the body
 of a try block, the throw continues up the call stack until it is in the body of
 an an enclosing try block, or the call stack is flushed. If the call stack is
-flushed, the host VM defines how to handle uncaught exceptions.  Otherwise, the
+flushed, the embedder defines how to handle uncaught exceptions.  Otherwise, the
 found enclosing try block is the catching try block.
 
 A throw inside the body of a catch block is never caught by the corresponding
 try block of the catch block, since instructions in the body of the catch block
 are not in the body of the try block.
 
-Once a catching try block is found for the thrown exception value, the operand
-stack is popped back to the size the operand stack had when the try block was
-entered, and then the values of the caught exception value is pushed onto the
-stack.
+As expected, it is left to the embedder to define how WebAssembly exceptions are
```

Comment:
This seems to repeat what is already said on L193.

Reply:
Deleted